src/app/rpm-ostree-countme.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Weekly rpm-ostree Count Me reporting
+Documentation=man:rpm-ostree-countme(1)
```

Comment:
I'm sure you're planning to do this already, but just so it's tracked: need man page for this.

Reply:
I wrote that but made the command hidden. I can do the man page but do we really need it?

Reply:
Maybe as a mention in the main manpage

Reply:
I think we should document the service/timer units somewhere. We could document it in the docs (`docs/`), though I suggested manpage because it's what we do for the automatic updates stuff, which is similar to this (systemd units driving a hidden knob) -- see `rpm-ostreed-automatic.xml`. Up to you! (Personally as a user, I would prefer man pages.)

Reply:
Will do man pages & docs.